datalad/cmdline/tests/test_main.py

```diff
@@ -211,8 +211,8 @@ def test_script_shims():
             with open(which.rstrip()) as f:
                 content = f.read()
         else:
-            from distutils.spawn import find_executable
-            content = find_executable(script)
+            from shutil import which
+            content = which(script)
```

Comment:
Cool!, I was not aware that `shutil` provides `which` on windows!

What would be the advantage of having non-windows be a special case now? `which` is not even a POSIX command, so using the `shutil` implementation should work more robustly in general.

Reply:
I was only looking at the `distutils` import when I made the change, not at the surrounding code.  I've added a commit to use `shutil.which()` on all platforms.

Reply:
Cool, thx!